ArmAngles arm_angle_initial
ArmPosition position
---
ArmAngles arm_angle_final